samplr/kubernetes/deployment.yaml

```diff
@@ -123,7 +123,7 @@ spec:
       labels:
         app: samplrd-sprvsr
     spec:
-      serviceAccountName: samplr-sprvsr-sa
+      serviceAccountName: maintner-sprvsr-sa
```

Comment:
Not a docs change, is this correct?

Reply:
This is correct; the prod cluster uses the same service account for samplr and maintner.